roles/eda/templates/eda.configmap.yaml.j2

```diff
@@ -87,10 +87,20 @@ data:
             }
 
             location / {
+                rewrite ^/$ /eda/ permanent; # Redirects all traffic from root to /eda/
+            }
+
+            location = /eda {
+                return 301 $scheme://$host/eda/;
+            }
+
+            location /eda/ {
                 autoindex off;
                 expires off;
                 add_header Cache-Control "public, max-age=0, s-maxage=0, must-revalidate" always;
-                try_files $uri /index.html =404;
+
+                # Serve /usr/share/nginx/html/index.html for requests to /eda/
```

Comment:
is this comment correct?

Reply:
According to the config above, it looks like it:
```
root /usr/share/nginx/html;
```

Reply:
It is, however, downstream the root path changes, so probably better to just leave it out anyways.